bai-tap-buoi-9/main.js

```diff
@@ -1 +1,151 @@
-const 
\ No newline at end of file
+// Hàm kiểm tra số nguyên
+function checkNumber(number) {
+    if (parseInt(number)==number) {
+        return true;
+    }
+    else {
+        return false;
+    }
+}
+// Bài 1:
+const realSalary = (money) => {
+    if (money >= 15000000) {
+        console.log('Thuế thu nhập: ' + money*0.3)
+        console.log('Lương ròng: '+money*0.7)
+    }
+    else if (money >= 7000000 && money < 15000000) {
+        console.log('Thuế thu nhập: ' + money*0.2)
+        console.log('Lương ròng: '+money*0.8)
+    }
+    else if (money >= 0 && money < 7000000) {
+        console.log('Thuế thu nhập: ' + money*0.1)
+        console.log('Lương ròng: '+money*0.9)
+    }
+    else {
+        console.log('Nhập đúng tiền lương')
+    }
+}
+
+// Bài 2:
+const ageSchool = (age) => {
+    if (checkNumber(age) && age >= 16) {
+        console.log('Bạn đã đủ tuổi vào lớp 10')
+    }
+    else if(checkNumber(age) && age<16) {
+        console.log('Bạn chưa đủ tuổi vào lớp 10')
+    }
+    else{
+        console.log('Mời bạn nhập số nguyên')
+    }
+}
+// Bài 3:
+const equalsNumber = (number) => {
+    if(checkNumber(number) && number>100) {
+        console.log('Số '+number+' lớn hơn 100')
+    }
+    else if(parseInt(number)==number && number==100) {
+        console.log('Hai số bằng nhau')
+    }
+    else if(parseInt(number)==number && number<100) {
+        console.log('Số '+number+' nhỏ hơn 100')
+    }
+    else {
+        console.log('Mời bạn nhập số nguyên')
+    }
+}
+
+// Bài 4: 
+const findMax = (a,b,c) =>{
+    var flag=0
```

Comment:
không được dùng `var`
`flag` là gì?

```suggestion
    // nếu 1 trong 3 không phải là số thì dừng luôn
    if (isNotANumber(a) || isNotANumber(b) || isNotANumber(c)) return;
  
    let max = a // max ban đầu là a
```